src/mtx3/commands/uplink/getCurrentStatusMeter.ts

```diff
@@ -0,0 +1,305 @@
+/**
+ * Uplink command to get current parameters.
+ *
+ * @packageDocumentation
+ *
+ * @example create command instance from command body hex dump
+ * ```js
+ * import * as getCurrentStatusMeter from 'jooby-codec/mtx3/commands/uplink/getCurrentStatusMeter.js';
+ *
+ * // simple response
+ * const bytes = [
+ *     0x00, 0x01, 0x22, 0x50, 0x00, 0x00, 0x87, 0x07, 0x00, 0x00, 0x30, 0x39, 0x00, 0x01, 0x09, 0x32,
+ *     0x00, 0x0e, 0x99, 0x36, 0x00, 0x00, 0x01, 0x54, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01, 0xb4,
+ *     0x01, 0x85, 0x01, 0x01, 0x01, 0x02, 0x03, 0x10, 0x01
+ * ];
+ *
+ * // decoded payload
+ * const parameters = getCurrentStatusMeter.fromBytes(bytes);
+ *
+ * console.log(parameters);
+ * // output:
+ * {
+ *     operatingSeconds: 74320,
+ *     tbadVAAll: 34567,
+ *     tbadVBAll: 12345,
+ *     tbadVCAll: 67890,
+ *     tbadIMAXAll: 956726,
+ *     tbadPMAXAll: 340,
+ *     tbadFREQ: 436,
+ *     relayStatus: true,
+ *     statusEvent: {
+ *         CASE_OPEN: true,
+ *         MAGNETIC_ON: false,
+ *         PARAMETERS_UPDATE_REMOTE: true,
+ *         PARAMETERS_UPDATE_LOCAL: false,
+ *         RESTART: false,
+ *         ERROR_ACCESS: false,
+ *         TIME_SET: false,
+ *         TIME_CORRECT: true,
+ *         DEVICE_FAILURE: false,
+ *         CASE_TERMINAL_OPEN: false,
+ *         CASE_MODULE_OPEN: false,
+ *         TARIFF_TABLE_SET: false,
+ *         TARIFF_TABLE_GET: true,
+ *         PROTECTION_RESET_EM: false,
+ *         PROTECTION_RESET_MAGNETIC: false
+ *     },
+ *     centerAlert: true,
+ *     calEnableFlag: true,
+ *     curTariff: {
+ *         consumedEnergies: 1,
+ *         maximumPowers: 2,
+ *         generatedEnergies: 3
+ *     },
+ *     isSummerTime: true
+ * }
+ * ```
+ *
+ * [Command format documentation](https://github.com/jooby-dev/jooby-docs/blob/main/docs/mtx3/commands/GetCurrentStatusMeter.md#response)
+ */
+
+import * as command from '../../../mtx/utils/command.js';
+import {READ_ONLY} from '../../../mtx/constants/accessLevels.js';
+import {IEventStatus, eventStatusMask} from '../../../mtx/utils/CommandBinaryBuffer.js';
+import CommandBinaryBuffer, {ICommandBinaryBuffer} from '../../utils/CommandBinaryBuffer.js';
+import * as bitSet from '../../../utils/bitSet.js';
+import * as types from '../../types.js';
+
+
+interface ITariff {
```

Comment:
rename fields
`
interface ITariffs {
    'A+': types.TUint8;

    maximumPowers: types.TUint8;

    /**
     * Tariff for the group of generated energies (`3rd` grid).
     */
     'A-': types.TUint8;
}

`

Reply:
applied